source/developers/how-tos/switch-to-the-sphinx-book-theme.rst

```diff
@@ -57,8 +57,6 @@ Steps
           "repository_url": TODO: Add a github URL Here, for example https://github.com/openedx/repo-name,
           "repository_branch": TODO: Add the correct branch, for example 'main',
           "path_to_docs": "docs/",
-          "logo_only": True,
-          "home_page_in_toc": True,
```

Comment:
Where can we learn more about what the supported options are? I didn't see their removal in the theme's changelog and I'm wondering where you're looking.

Reply:
So the latest settings are here: https://sphinx-book-theme.readthedocs.io/en/stable/reference.html

The the settings come through the HTML templates for the theme so it got missed in the changelog and I had to find it much the same way I think you did but I forgot to update the How-to then when we fixed forward in the existing places.

Reply:
Ah, oh well.